docs/news.rst

```diff
@@ -3,6 +3,132 @@
 Release notes
 =============
 
+.. _release-2.3.0:
+
+Scrapy 2.3.0 (2020-0?-??)
+-------------------------
+
+Highlights:
+
+* :ref:`Feed exports <topics-feed-exports>` support :ref:`Google Cloud
+  Storage <topics-feed-storage-gcs>`
+* New :setting:`FEED_EXPORT_BATCH_ITEM_COUNT` setting for batch deliveries
```

Comment:
What do you think about explaining this a bit more? E.g. "previously, Scrapy's Feed Exports were exporting items to S3 (and ...?) at the end of the crawl. With FEED_EXPORT_BATCH_ITEM_COUNT option it is now possible to export items to cloud providers during the crawl, in batches of a specified size."